lib/request.js

```diff
@@ -58,11 +58,14 @@ var request = function (params, body) {
       return err;
     })
     .then(function (response) {
+      var contentType = response.headers['content-type'] || [];
+      var requestId = response.headers['X-Request-ID'] || null;
```

Comment:
The console messages are actually helpful. Thanks for fixing. Pushed a commit to fix the goof up.